### DISCUSSION TITLE
发帖前你需要知道的事

### DISCUSSION THREAD

**Discussion by 0xxb** [lsky-org/lsky-pro#350] [公告]
## 👋  
- 如果你正准备提问，寻求帮助，第一步应该查找文档、issues 列表，提问前请先阅读 [提问的智慧](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)
- 有想法、建议，发帖前请思考是否合理。
- 非常欢迎你分享任何关于兰空图床的使用心得或相关教程。

**Comment by akatsukiro** [comment]
第一条加个Discussions应该会更完善，我觉得。不知道您意下如何。

**Reply by 0xxb** [reply]
我觉得这句话已经够明确的了。

**Reply by akatsukiro** [reply]
嗯，确实是。
